projects/addon-doc/components/navigation/navigation.component.ts

```diff
@@ -228,5 +230,11 @@ export class TuiDocNavigationComponent {
             inline: 'nearest',
             behavior: this.scrollBehavior,
         });
+
+        this.ngZone.runOutsideAngular(() => {
+            setTimeout(() => {
+                element.classList.remove('tui-doc-animated-example');
```

Comment:
I wonder if we can use `anchorScrolling: 'enabled'` in `RouterModule.forRoot(...)` instead of our own implementation of scrolling to the example.

Also perhaps we can use `:target` CSS selector to apply animation instead of manual class management?

Reply:
<img width="479" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/270b4aba-d49e-4128-b16a-f166f63f3e5c">

It doesn't work ((

Reply:
Ok, and `:target` would trigger shaking animation when we click on copy link. Ignore this comment then. But why do you need to skip scroll when example is animated?

Reply:
Sorry, it's my fault, anchorScrolling works well

Reply:
fixed